features/keychain/module/__tests__/binauth.test.js

```diff
@@ -1,6 +1,6 @@
 import { mnemonicToSeed } from 'bip39'
 
-import { EXODUS_KEY_IDS } from '../key-identifier'
+import { EXODUS_KEY_IDS } from '@exodus/key-ids'
```

Comment:
shouldn't we change this when we actually remove the functionality? Otherwise tests and local code might have a mismatch

Reply:
the existing ones will never change and new ones should no longer be added here though 🤔

Reply:
do we use this in lots of places? maybe just cut the cord and make the breaking change?

Reply:
> maybe just cut the cord and make the breaking change?

not many places in the platforms, but a few packages we have to make sure are updated before or at the same time as keychain: https://github.com/ExodusMovement/exodus-hydra/pull/5665

I am happy with removing them - @kewde?

Reply:
removed in https://github.com/ExodusMovement/exodus-oss/pull/52/commits/f358690b4e47cfc8747082905c0b2b150768399a and marked PR as breaking